main.go

```diff
@@ -21,29 +21,22 @@ import (
 	"os"
 
 	"go.uber.org/zap/zapcore"
-	corev1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/labels"
+	v1 "k8s.io/api/apps/v1"
```

Comment:
- I don't see this being part of current commit
- Nevertheless, it's better to use `corev1` itself

Reply:
Updated in latest commit